data_steward/tools/run_deid.py

```diff
@@ -26,11 +26,11 @@
 DEID_TABLES = [
     'person', 'observation', 'visit_occurrence', 'condition_occurrence',
     'drug_exposure', 'procedure_occurrence', 'device_exposure', 'death',
-    'measurement', 'location', 'care_site', 'specimen', 'observation_period'
+    'measurement', 'location', 'specimen', 'observation_period'
```

Comment:
@dcampbell-vumc I think the scope is wrong on this ticket.  You should leave `care_site` as a `DEID_TABLES` item.  Just add it to the `SUPPRESSED_TABLES` list.